R/apply-pgs.R

```diff
@@ -21,7 +21,7 @@ validate.vcf.input <- function(vcf.data) {
 
     }
 
-validate.pgs.data.input <- function(pgs.weight.data, use.external.effect.allele.frequency) {
+validate.pgs.data.input <- function(pgs.weight.data, use.external.effect.allele.frequency, correct.strand.flips, remove.ambiguous.allele.matches, remove.mismatched.indels) {
```

Comment:
These boolean flag arguments are a good opportunity to add clarity and establish default functionality. Just adding `TRUE` or `FALSE` defaults might go a long way here to implying the primary use case(s).